webpack/ForemanInventoryUpload/SubscriptionsPageExtension/InventoryAutoUpload.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Switch, Level, LevelItem } from '@patternfly/react-core';
+import { OverlayTrigger, Tooltip, Icon, noop, FormGroup, Grid } from 'patternfly-react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Col, ControlLabel } from 'react-bootstrap';
+
+export const InventoryAutoUploadSwitcher = (props) => {
+  return (
+    <FormGroup>
+      <Grid>
+        <h3>{__('Red Hat Cloud Inventory')}</h3>
+        <hr />
+        <Grid.Row>
+          <div id="simple-content-access">
+            <Col sm={5}>
+              <ControlLabel className="control-label" style={{ paddingTop: '0' }}>
+                <Level>
+                  <LevelItem>
+                    <span className="sca-label">{__('Inventory Auto Upload')}</span>
```

Comment:
```suggestion
                    <span className="rh-cloud-inventory-auto-upload-label">{__('Inventory Auto Upload')}</span>
```
sca is Simple Content Access